test/db/formats/luac/luac

```diff
@@ -0,0 +1,134 @@
+EOF
+RUN
+
+NAME=LUAC: symbols
+FILE=bins/luac/none.luac
+CMDS=is~UPVALUE
+EXPECT=<<EOF
+0   0x0000003a 0x0000003a NONE UPVALUE 3        @none.lua_upvalue__ENV
+EOF
+RUN
+
+NAME=LUAC: entry
+FILE=bins/luac/none.luac
+CMDS=ie~program
+EXPECT=<<EOF
+vaddr=0x00000030 paddr=0x00000030 haddr=-1 type=program
+EOF
+RUN
+
+NAME=LUAC: big file
+FILE=bins/luac/big.luac
+CMDS=iS~header
+EXPECT=<<EOF
+0   0x00000020  0xe44 0x00000020  0xe44 -r-- learnlua.lua.header
+6   0x000007ab   0x70 0x000007ab   0x70 -r-- fcn.000007ab.header
+12  0x0000081a   0x52 0x0000081a   0x52 -r-- fcn.0000081a.header
+18  0x00000830   0x31 0x00000830   0x31 -r-- fcn.00000830.header
+24  0x0000086b   0x6b 0x0000086b   0x6b -r-- fcn.0000086b.header
+30  0x000008d5   0x27 0x000008d5   0x27 -r-- fcn.000008d5.header
+36  0x000008fb   0x27 0x000008fb   0x27 -r-- fcn.000008fb.header
+42  0x00000921   0x44 0x00000921   0x44 -r-- fcn.00000921.header
+48  0x00000964   0x44 0x00000964   0x44 -r-- fcn.00000964.header
+54  0x000009a7   0x3e 0x000009a7   0x3e -r-- fcn.000009a7.header
+60  0x000009e4   0xa6 0x000009e4   0xa6 -r-- fcn.000009e4.header
+66  0x00000a89   0x83 0x00000a89   0x83 -r-- fcn.00000a89.header
+72  0x00000b0b   0x54 0x00000b0b   0x54 -r-- fcn.00000b0b.header
+78  0x00000b5e   0x6b 0x00000b5e   0x6b -r-- fcn.00000b5e.header
+84  0x00000bc8   0x71 0x00000bc8   0x71 -r-- fcn.00000bc8.header
+EOF
+RUN
+
+NAME=LUAC: big file string symbol loading
+FILE=bins/luac/big.luac
+CMDS=is~twas
+EXPECT=<<EOF
+0   0x000005d8 0x000005d8 NONE CONST_STRING 10       @learnlua.lua_const_twas false
+EOF
+RUN
+
+NAME=LUAC: big file num symbol loading
+FILE=bins/luac/big.luac
+CMDS=is~1729,1.21
+EXPECT=<<EOF
+0   0x00000687 0x00000687 NONE CONST_NUM    8        @learnlua.lua_const_1729
+0   0x000006d5 0x000006d5 NONE CONST_NUM    8        @learnlua.lua_const_1.210000
+EOF
+RUN
+
+NAME=LUAC: big file symbols
+FILE=bins/luac/big.luac
+CMDS=is~CONST_NUM
+EXPECT=<<EOF
+0   0x000004db 0x000004db NONE CONST_NUM    8        @learnlua.lua_const_42
+0   0x00000590 0x00000590 NONE CONST_NUM    8        @learnlua.lua_const_5
+0   0x000005fb 0x000005fb NONE CONST_NUM    8        @learnlua.lua_const_0
+0   0x00000687 0x00000687 NONE CONST_NUM    8        @learnlua.lua_const_1729
+0   0x00000690 0x00000690 NONE CONST_NUM    8        @learnlua.lua_const_6.280000
+0   0x000006d5 0x000006d5 NONE CONST_NUM    8        @learnlua.lua_const_1.210000
+0   0x000006ed 0x000006ed NONE CONST_NUM    8        @learnlua.lua_const_1
+0   0x000006f6 0x000006f6 NONE CONST_NUM    8        @learnlua.lua_const_2
+0   0x00000703 0x00000703 NONE CONST_NUM    8        @learnlua.lua_const_3
+EOF
+RUN
+
+NAME=LUAC: disasm crash
+FILE=bins/luac/big.luac
+CMDS=pd 10 >/dev/null
+EXPECT=<<EOF
+EOF
+RUN
+
+NAME=LUAC: broken file
+FILE=bins/luac/random.luac
+CMDS=<<EOF
+EOF
+EXPECT=<<EOF
+EOF
+EXPECT_ERR=<<EOF
+Bad Luac File : Truncated Code at [0x2f]
+EOF
+RUN
+
+NAME=LUAC: iz strings
+FILE=bins/luac/big.luac
+CMDS=iz~?print
```

Comment:
I suggest you always use
```
CMDS=<<EOF
EOF
```

Reply:
Yeah, but there seems no way to put `=<<EOF` in [assembly test](https://github.com/rizinorg/rizin/tree/dev/test#writing-assembly-tests)

For other cmd tests I will refactor them as you suggest :).

Reply:
Also noted that the internal object is `bin_obj_info` in `load_buffer: line 56` : 
```
	// how to free it ? in .finit?
	bin_info_obj = luac_build_info(proto);
	if (bin_info_obj == NULL) {
		lua_free_proto_entry(proto);
		return false;
	}
```

Reply:
No need for the assembly tests, just for the others.